app.js

```diff
@@ -5,7 +5,7 @@ var cookieParser = require('cookie-parser');
 var logger = require('morgan');
 
 var indexRouter = require('./routes/index');
-var usersRouter = require('./routes/users');
+var lottoRouter = require('./routes/lotto');
```

Comment:
```suggestion
const lottoRouter = require('./routes/lotto');
```
var 대신 const or let으로